src/runtime-rs/Makefile

```diff
@@ -78,18 +320,33 @@ define INSTALL_FILE
 	install -D -m 644 $1 $(DESTDIR)$2/$1 || exit 1;
 endef
 
+define INSTALL_CONFIG
+	sudo install --mode 0644 -D $1 $(DESTDIR)$2/$(notdir $1);
```

Comment:
As mentioned above, please use `utils.mk` for installation. It hasn't landed yet (please tal all! ;) but I've spent **a lot** of time working on fixing the install issues and removing all the duplication on https://github.com/kata-containers/kata-containers/pull/3905, so please use that if at all possible.

Reply:
Thanks. I will use that once it has landed.